lib/screens/v2/widget_instance/common/base_alert.ex

```diff
@@ -236,23 +200,68 @@ defmodule Screens.V2.WidgetInstance.Common.BaseAlert do
 
   defp get_location_atom(_, _, _), do: :elsewhere
 
+  # Only used to decide "takeover_alert" for bus shelter, bus einks, and prefare screens
   def informs_all_active_routes_at_home_stop?(t) do
-    MapSet.subset?(active_routes_at_stop(t), informed_routes_at_home_stop(t))
+    MapSet.subset?(active_routes_at_stop(t), MapSet.new(informed_routes_at_home_stop(t)))
   end
 
   @spec active_routes_at_stop(t()) :: MapSet.t(route_id())
-  def active_routes_at_stop(t) do
-    t
-    |> SAW.routes_at_stop()
+  def active_routes_at_stop(%{location_context: %{routes: routes}}) do
```

Comment:
Hmm, ok... though if the reason for not using a helper function is performance, I wouldn't worry about it. It's a minuscule amount of work to do that mapping.

I would disagree if the argument is about ergonomics as well, because BaseAlert already has other helper functions that do similar basic "accessing" of values, e.g. [`informed_entities`](https://github.com/mbta/screens/blob/bde871e76870f24cca7e19c0292260d1d306c748/lib/screens/v2/widget_instance/common/base_alert.ex#L82).

If you still want to keep the duplicate data fields, then ok, but I definitely prefer merging them.

Reply:
Since the helper would be "get route ids from a list of routes", I think that would better live in `Routes`, which means that module would need to be imported to other modules where it isn't currently needed. 

I guess I'll do the merging and see how it feels.